src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+import java.util.Random;
+public class Lottery {
+    public static Ball getRandomBall(){
```

Comment:
Don't use static methods in your solution. It's better to create an instance of a class whose method you want to call.